applications/GeoMechanicsApplication/python_scripts/geomechanics_analysis.py

```diff
@@ -57,14 +57,15 @@ def _CalculateTotalDisplacement(self,node, old_total_displacement):
 
     def ResetIfHasNodalSolutionStepVariable(self, variable):
         if self._GetSolver().main_model_part.HasNodalSolutionStepVariable(variable):
-            KratosMultiphysics.VariableUtils().SetHistoricalVariableToZero(variable, self._GetSolver().GetComputingModelPart().Nodes)
-            for node in self._GetSolver().GetComputingModelPart().Nodes:
-                new_value = node.GetSolutionStepValue(variable, 0)
-                node.SetSolutionStepValue(variable, 1, new_value)
-
-    def ModifyInitialGeometry(self):
-        # Overrides the base class. Necessary to let reset_displacements function correctly i.c.w. prescribed displacements/rotations.
-        # The reset needs to take place befor the Initialize of the processes, as these will set the Dirichlet condition.
+            zero_vector = Kratos.Array3([0.0, 0.0, 0.0])
```

Comment:
I believe you can also write it like this:
```suggestion
            zero_vector = variable.Zero()
```

I'm not sure whether the call operator `()` is needed here.

Reply:
Unfortunately it doesn't work (with or without parentheses): `'Kratos.Array1DVariable3' object has no attribute 'Zero'`

Reply:
We found that if we export the function to python it does work, but that'd require a core change, which we don't want to do right now